Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml

```diff
@@ -97,4 +97,16 @@
 - type: entity
   id: ClosetRadiationSuitFilled
   parent: ClosetRadiationSuit
-  suffix: Filled
\ No newline at end of file
+  suffix: Filled
+  description: "More comfortable then radiation poisioning."
```

Comment:
Not sure if mistake, or implying that the rad suits do nothing so you might as well get comfortable first.

Reply:
I imagine that radiation suits are not very comfortable to wear. None the less that's better (more comfortable) then not wearing them and getting radiation poison.

Reply:
then -> than

Reply:
Ah gotcha, I'll push a correction to the spelling.